packages/gatsby-monorepo-plugin/package.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "@manypkg/gatsby-monorepo-plugin",
```

I think this is the convention

```suggestion
    "name": "@manypkg/gatsby-plugin-monorepo",
```